app/models/relationship.rb

```diff
@@ -0,0 +1,4 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```

Comment:
```suggestion
  belongs_to :follower, class_name: User.name
```
sửa tương tự các chỗ khác